examples/magic-link-with-jwt/.gitignore

```diff
@@ -0,0 +1,307 @@
+node_modules
```

Comment:
Please keep this to the bear minimum like we have in the `.gitignore` file in the templates

https://github.com/remix-run/remix/blob/main/templates/remix-ts/.gitignore